src/pages/con/2021/speakers/index.tsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
+import { Helmet } from 'react-helmet';
 import { PageProps } from 'gatsby';
 import Layout from '@con/components/2021/layout';
 import SpeakersPage from '@con/components/common/SpeakersPage';
 
 const Speakers: React.ComponentType<PageProps> = () => (
   <Layout logoAlwaysVisible>
+    <Helmet>
+      <title>Speakers</title>
```

Comment:
Attention, c'est le même title que pour cette année. Il faudrait ajouter l'année.
Je propose : `Speakers of the 2021`